packages/fx-core/src/core/middleware/projectMigrator.ts

```diff
@@ -131,7 +131,7 @@ class EnvConfigName {
 export class ArmParameters {
   static readonly FEStorageName = "frontendHostingStorageName";
   static readonly IdentityName = "userAssignedIdentityName";
-  static readonly SQLServer = "azureSqlServerName";
+  static readonly SQLServer = "sqlServerName";
   static readonly SQLDatabase = "azureSqlDatabaseName";
```

Comment:
azureSqlDatabaseName should be sqlDatabaseName

Reply:
already updated.